### PR TITLE
arcgis rest servers

### PR DESCRIPTION
**Pull request by felixfeckler**
Started to work on #5290 and realized that there is no real documentaion
for the ArcGIS REST Servers. So i first added the Table in
https://docs.qgis.org/testing/en/docs/user_manual/introduction/browser.html?highlight=arcgis%20rest%20servers#tiles-and-web-services
and updated an image.

My idea is to document it in the https://docs.qgis.org/testing/en/docs/user_manual/introduction/browser.html?highlight=arcgis%20rest%20servers#tiles-and-web-services
section, in the subsection "Connectiong to web services" as a new item.

Thoughts?

Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

**Comment by DelazJ** [comment]
Hi @felixfeckler 
> My idea is to document it in the https://docs.qgis.org/testing/en/docs/user_manual/introduction/browser.html?highlight=arcgis%20rest%20servers#tiles-and-web-services
section, in the subsection "Connectiong to web services" as a new item.
>
> Thoughts?

I asssume the idea is to document how to load ArcGIS REST layers and all the particular settings that requires. I'd then instead see it in https://docs.qgis.org/testing/en/docs/user_manual/managing_data_source/opening_data.html#connecting-to-web-services, next to the other providers.

**Comment by felixfeckler** [comment]
Hey @DelazJ 

Well.... you are right, and to be honest, i messed up the link and wanted to put it in the exact same section you suggested and instead pasted two times the same link...

**Comment by DelazJ** [comment]
> i messed up the link and wanted to put it in the exact same section you suggested and instead pasted two times the same link...

This was also my thought when I realized how close the URLs were 😄

**Comment by felixfeckler** [comment]
Okay but so we agree on where to document it :)

**Comment by DelazJ** [comment]
Something you are willing to do in this branch or in another one?

**Comment by felixfeckler** [comment]
Sorry read it too late, just opened another one.

File: docs/user_manual/introduction/browser.rst [hunk 1]
```diff
@@ -305,17 +305,21 @@ Tiles and Web Services
...
-|               | :guilabel:`Edit…` connection settings        | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     |             | |checkbox| |
+|               | :guilabel:`Edit…` connection settings        | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     | |checkbox|  | |checkbox| |
 |               +----------------------------------------------+------------+-------------------+------------+------------+----------------+-------------+------------+
-|               | :guilabel:`Delete` connection                | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     |             | |checkbox| |
+|               | :guilabel:`Delete` connection                | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     | |checkbox|  | |checkbox| |
+|               +----------------------------------------------+------------+-------------------+------------+------------+----------------+-------------+------------+
+|               | :guilabel:`View Service Info` in Webbrowser  |            |                   |            |            |                | |checkbox|  |            |
```
Comment (DelazJ) [inline docs/user_manual/introduction/browser.rst:314]:
```suggestion
|               | :guilabel:`View Service Info` in Web browser |            |                   |            |            |                | |checkbox|  |            |
```

File: docs/user_manual/introduction/browser.rst [hunk 2]
```diff
@@ -305,17 +305,21 @@ Tiles and Web Services
...
+|               | :guilabel:`Add layer to Project`             | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     | |checkbox|  | |checkbox| |
 |               +----------------------------------------------+------------+-------------------+------------+------------+----------------+-------------+------------+
-|               | :guilabel:`Add layer to Project`             | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     |             | |checkbox| |
+|               | Open :guilabel:`Layer properties…` dialog    | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     | |checkbox|  | |checkbox| |
 |               +----------------------------------------------+------------+-------------------+------------+------------+----------------+-------------+------------+
-|               | Open :guilabel:`Layer properties…` dialog    | |checkbox| | |checkbox|        | |checkbox| | |checkbox| | |checkbox|     |             | |checkbox| |
+|               | :guilabel:`View Service Info` in Webbrowser  |            |                   |            |            |                | |checkbox|  |            |
```
Comment (DelazJ) [inline docs/user_manual/introduction/browser.rst:322]:
```suggestion
|               | :guilabel:`View Service Info` in Web browser |            |                   |            |            |                | |checkbox|  |            |
```

**Review by DelazJ — COMMENTED** [review]
Typo?

**Pull request merged**